dsl/read.go

```diff
@@ -57,14 +66,20 @@ func ReadHopsFileContents(hopsFileContent []FileContent) (*hcl.BodyContent, stri
 
 	// parse the hops files
 	for _, file := range hopsFileContent {
+		// Add all file contents to the hash
+		sha1Hash.Write(file.Content)
```

Comment:
I actually mucked up here. This should have been a sha256 not sha1.

It's not a _huge_ deal, but does annoy me. Might be worth changing now before it's a big deal?

Reply:
This will break all previous hashes. Same as above.

Changed. Uses sha256